### PR TITLE
Update enter-business-information.md

### LINKED ISSUE #12450: Bookings Calendar is now Bookings Page
Microsoft has changed terminology. what used to be called a booking calendar is now called a booking page. the only thing still referred to as a booking calendar is the actual calendar in the bookings application. Changing references to booking calendar to booking page is advisable everywhere. 

If anyone knows how to get ahold of folks at microsoft. The confirmation email that is sent out when a booking page is created is still using the outdated booking calendar terminology, too.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: db93982a-4d08-b3be-4f3d-33ff5b9d20b2
* Version Independent ID: 30541004-90bb-b9fc-02a6-a04e7a921b13
* Content: [Enter your Bookings business information](https://learn.microsoft.com/en-us/microsoft-365/bookings/enter-business-information?view=o365-worldwide)
* Content Source: [microsoft-365/bookings/enter-business-information.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/bookings/enter-business-information.md)
* Service: **bookings**
* GitHub Login: @kwekuako
* Microsoft Alias: **kwekua**

### PR DESCRIPTION
**Pull request by msbemba**
Updated bookings calendars to booking page

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12450

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/bookings/enter-business-information.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The information you provide here will be displayed on the page customers and cli
 
 1. In the navigation pane, select **Your calendar** > **Business information** in the left pane.
 
-1. On the **Basic details** section, enter your business name, address, and phone number you would like to use for your Bookings calendar.
+1. On the **Basic details** section, enter your business name, address, and phone number you would like to use for your Bookings Page.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/enter-business-information.md]:
```suggestion
1. On the **Basic details** section, enter your business name, address, and phone number you would like to use for your booking page.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder

**Pull request merged**